db/migrate/20230506213710_create_topics.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTopics < ActiveRecord::Migration[7.0]
+  def change
+    create_table :topics do |t|
+      t.string :title
```

Comment:
I think you should add some constraints, for example, the title should be unique and not null.